javascript-practice/src/js/helpers/validation.js

```diff
@@ -1,7 +1,24 @@
 import { REGEX } from '../constants/regex';
 
+validatePhoneNumber = (phoneNumber) => {
+  return REGEX.PHONE_NUMBER.test(phoneNumber);
+};
+
 validatePassword = (password) => {
   return REGEX.PASSWORD.test(password);
 };
 
-export { validatePassword };
+validateConfirmPassword = (password, confirmPassword) => {
+  return password === confirmPassword;
+};
+
+validateInputLength = (value) => {
+  return value.length > 2;
```

Comment:
Why input length is fixed for 2? If I have more?

Reply:
i done fix